simulation-gazebo.md

```diff
@@ -97,7 +97,7 @@ ______  __   __    ___
 | |     / /^\ \ \___  |
 \_|     \/   \/     |_/
 
-mainapp starting.
+px4 starting.
```

Comment:
And this can just be `starting`.

Sorry, these changes are actually needed on the Firmware side.